src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_release.kt

```diff
@@ -200,7 +201,7 @@ class OffenderResourceIntTest_release : ResourceTest() {
         ).isBadRequest
           .expectBody()
           .jsonPath("userMessage")
-          .isEqualTo("Prisoner is not currently active")
+          .isEqualTo("Booking $bookingId is not active")
```

Comment:
A few error messages have changed - rather than using bespoke errors in the old service we are now using common errors also used in other services.